config/routes.rb

```diff
@@ -3,10 +3,16 @@
 
   resources :repos do
     get "/cohort/:cohort_id", to: "repos#show", as: :cohort
+    get "/student/:student_id/feedback/new", to: "feedback#new", as: :new_feedback
+    post "/student/:student_id/feedback/", to: "feedback#create", as: :feedback
```

Comment:
I believe these two `get` and `post` directives are redundant. We should change the following code (on [Line 8](https://github.com/Ada-Developers-Academy/ada-prs/pull/4/files#diff-21497849d8f00507c9c8dcaf6288b136R8)):
```ruby
resources :students do
  resources :feedback
```

to

```ruby
resources :students, only: [] do
  resources :feedback, only: [:new, :create]
end
```
and also change [Line 27 in `app/views/repos/show.html.erb`](https://github.com/Ada-Developers-Academy/ada-prs/pull/4/files#diff-f6df129df44f0b68f11aa859037ae2f3R27) to use `new_repo_student_feedback_path`.